azkaban-common/src/main/java/azkaban/alert/Alerter.java

```diff
@@ -33,7 +34,13 @@ public interface Alerter {
 
   void alertOnFirstError(ExecutableFlow exflow) throws Exception;
 
-  void alertOnSla(SlaOption slaOption, String slaMessage) throws Exception;
+  @Deprecated
+  default void alertOnSla(SlaOption slaOption, String slaMessage) throws Exception { }
```

Comment:
I know this is in your interface and all, but is there anyway we can delete this? Is it used elsewhere?

Reply:
After some time we can definitely delete these deprecated APIs to keep the Alerter interface clean. For now we're deprecating and defaulting to old behavior to avoid breaking any existing custom alerters which have to implement the Alerter interface.